src/Sarif.UnitTests/Converters/AndroidStudioConverterTests.cs

```diff
@@ -251,8 +251,14 @@ public void AndroidStudioConverter_ConvertSarifResult_RecordsModuleAsTopLevelIfP
 
             LocationInfo locationInfo = GetLocationInfoForBuilder(builder);
 
-            locationInfo.Location.Should().Be(expectedLocation);
-            locationInfo.LogicalLocationComponents.SequenceEqual(expectedLogicalLocationComponents).Should().BeTrue();
+            LocationEqualityComparer.Instance.Equals(locationInfo.Location, expectedLocation)
+                .Should().BeTrue();
+
```

Comment:
In future, will be `locationInfo.Location.EqualsByValue(expectedLocation)` or something like that.